app/controllers/rooms_controller.rb

```diff
@@ -2,6 +2,11 @@ class RoomsController < ApplicationController
 
   before_action :authenticate_user!
 
+  def index
+    user = User.find(current_user.id)
+    @rooms = user.rooms
```

Comment:
Remove lines 6 and 7 because you can use the current user devise helper in your views.